packages/env/src/index.ts

```diff
@@ -1,12 +1,15 @@
 import path from 'path'
+import { fileURLToPath } from 'url'
 
 import dotenv from 'dotenv'
 
+const DIRNAME = path.dirname(fileURLToPath(import.meta.url))
```

Comment:
`__dirname` is not giving the right path when the package is used in `web` or `api`

Reply:
makes sense